main.cpp

```diff
@@ -46,7 +46,7 @@ inline bool plotly_script_exists () {
 
 int main(int argc, char *argv[])
 {
-	//testing
+
```

Comment:
Please don't leave useless empty lines!

Reply:
ok. I will do it but in future
let's automize this process with npm fromat
using c-lang style